src/include/printer/connectionwizard.rb

```diff
@@ -1089,7 +1089,11 @@ def changeSettingsDialog(selected)
             # ( hcitool scan | grep '...' ) & sleep 10 ; kill -9 $!
             # would kill only the sub shell.
             if !Printerlib.ExecuteBashCommand(
-                "hcitool scan >/tmp/hcitool_scan.out & sleep 10 ; kill -9 $! ; grep '..:..:..:..:..:..' /tmp/hcitool_scan.out | tr -s ' ' ; rm -f /tmp/hcitool_scan.out"
+                 "hcitool scan >/tmp/hcitool_scan.out & sleep 10" +
```

Comment:
please use a non-deterministic temp file path

Reply:
Fixed.

Now using Ruby's `Dir.mktmpdir` which not only creates a directory with an unpredictable name, but also sets it to permissions 0700.